staging/src/k8s.io/client-go/tools/cache/the_real_fifo.go

```diff
@@ -254,6 +260,12 @@ func (f *RealFIFO) Pop(process PopProcessFunc) (interface{}, error) {
 	return Deltas{item}, err
 }
 
+func (f *RealFIFO) Bookmark(resourceVersion string) error {
+	f.lock.Lock()
+	defer f.lock.Unlock()
+	return f.addToItems_locked(Bookmark, false, resourceVersion)
```

Comment:
we *do* want to skip transform for bookmark events

```suggestion
	return f.addToItems_locked(Bookmark, true, resourceVersion)
```